RiyazRizvi_Transcription_prac_61.py

```
@@ -0,0 +1,19 @@
+from fastapi import FastAPI,UploadFile,File
+import shutil
+from pathlib import Path
+
+app=FastAPI()
+
+@app.post('/UploadFile/')
+async def file(file:UploadFile=File(...)):
```

Ruff (0.13.1)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)